src/server/src/entities/job.ts

```diff
@@ -0,0 +1,78 @@
+// import mongoose, { Schema, Document } from 'mongoose';
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Job {
+    constructor(id: number, targetYears: string[], hoursPerWeek: number, description: string,
+        expirationDate: Date, startDate: Date, endDate: Date, type: string[], title: string, status: string,
+        minSalary: number, maxSalary: number, departmentId: string) {
```

Comment:
Here departmentId is a string because, for now, we will store the name of the department right?

Reply:
Yes.